.devcontainer/Dockerfile

```diff
@@ -35,7 +34,7 @@ ENV MAGMA_ROOT=/workspaces/magma
 ENV BUILD_TYPE=Debug
 ENV C_BUILD=/workspaces/magma/build/c
 ENV DEBIAN_FRONTEND=noninteractive
-ENV PYTHONPATH=$MAGMA_ROOT/lte/gateway/python/:$MAGMA_ROOT/orc8r/gateway/python/
```

Comment:
Deleted, because it is handled in bazel build directly now (see https://github.com/magma/magma/commit/bb2244a642ab34263ab3bcfdad353bf90469c730)